Dockerfile

```diff
@@ -1,6 +1,6 @@
 # specifying the platform here allows builds to work
 # correctly on Apple Silicon machines
-FROM --platform=amd64 ruby:2.7.4-slim-buster as base
+FROM --platform=amd64 ruby:2.7-slim-buster as base
```

Comment:
By not specifying the Z does that mean that we let it be whatever is "latest" and thus our builds may actually change from one run to the next? And if so, is that loose-ness intentional/desired?

Reply:
i was feeling shaky on bumping the base image to begin with. then you asked about it and i decided to try just updating the offending packages and that seems to have worked.